awx/api/serializers.py

```diff
@@ -64,6 +64,7 @@
 from awx.main.models.rbac import (
     get_roles_on_resource, role_summary_fields_generator
 )
+from awx.main.utils.common import model_to_dict
```

Comment:
I think this is now no longer used.

Reply:
It is being used in line 831.

Reply:
That is likely going to have the same objection then as the other usage of it -- it is returning too much info.

Reply:
I wonder if this would make more sense to expose as a related URL instead of as an actual field on the job record.

Reply:
The UI needs to construct a route around the info from it.

Reply:
Well, ok, maybe a summary field _would_ be the thing to do.

Reply:
Working on it.

Reply:
relevant comment: https://github.com/ansible/awx/pull/9201/files#r595450489

Reply:
With the summary field added, we shouldn't need this anymore.

Reply:
I have removed it.